### DISCUSSION TITLE
Welcome to BusinessCalculator Discussions!

### DISCUSSION THREAD

**Discussion by pranav627** [pranav627/BusinessCalculator#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shilpa-k-h** [comment]
need to add multiple fields functionality in expenses tab so that you can add multiple expenses in a business.

**Comment by shilpa-k-h** [comment]
having a central table with ranking most profitable business will make a great dashboard
